CanHazFunny/CanHazFunny/OutputService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CanHazFunny;
+
+public class OutputService : IPrintJokeInterface
```

Comment:
```suggestion
public class OutputService : IPrintJoke
```
Because it's prefixed with an `I` we know it's an interface and saying interface is redundant